docs-2.0/4.deployment-and-installation/5.zone.md

```diff
@@ -1,43 +1,55 @@
-# Manage Zone
+# Manage Zones
+
+!!! enterpriseonly
+
+    This feature is only available in the NebulaGraph Enterprise Edition.
 
 The Zone is a logical rack of storage nodes in a NebulaGraph cluster. It divides multiple storage nodes into manageable logical areas to achieve resource isolation. At the same time, you can control the Graph service to access the replica data in the specified Zone to reduce traffic consumption and improve access efficiency. This article describes how to use the Zone feature.
 
 ## Principle
 
-Storage nodes can be added to a Zone. When creating a graph space, you can specify a list of Zones. The graph space is created and stored on the storage nodes added in these Zones. Partition replicas are evenly stored in the Zones, as shown in the following figure.
+Within NebulaGraph, you can set up multiple Zones, with each Zone containing one or more Storage nodes. When creating a graph space, the system automatically recognizes these set Zones and stores graph space data on the Storage nodes within these Zones.
+
+It's important to note that when creating a graph space, you need to specify the number of replicas for data partitioning. At this point, **the specified number of partition replicas and the number of set Zones must be equal, otherwise, the graph space cannot be created**. This is because NebulaGraph evenly stores each replica of the graph space data across these set Zones.
+
+Since each Zone contains a complete set of graph space data partitions, at least one Storage node is required within each Zone to store these data partitions.
+
+The partition replicas in NebulaGraph achieve strong consistency through the [Raft](../1.introduction/3.nebula-graph-architecture/4.storage-service.md#raft_1) protocol. It's recommended to use an odd number of partition replicas, and therefore, it's also suggested to set an odd number of Zones.
+
+Taking the following picture as an example, when creating a graph space, the data is partitioned into 3 partitions, with 3 replicas for each partition, and the number of Zones is also 3. Six machines starting the Storage service are paired up and added to these 3 Zones. When creating the graph space, NebulaGraph evenly stores each replica of the graph space data across zone1 to zone3, and each Zone contains a complete set of graph space data partitions (part_1 to part_3).
```

Comment:
`Six machines starting the Storage ...` -> `Six machines hosting the Storage service...`